service/history/workflow_task_handler.go

```diff
@@ -215,6 +215,12 @@ func (handler *workflowTaskHandlerImpl) handleCommand(
 	command *commandpb.Command,
 	msgs *collection.IndexedTakeList[string, *protocolpb.Message],
 ) (*handleCommandResponse, error) {
+
+	handler.metricsHandler.Counter(metrics.CommandCounter.GetMetricName()).Record(
+		1,
+		metrics.NamespaceTag(handler.mutableState.GetExecutionInfo().NamespaceId),
+		metrics.CommandTypeTag(command.GetCommandType().String()))
```

Comment:
After some consideration, I decided to put this here, even though this will now include "unknown command types" (see default case statement). Which is actually a plus in my book since now we get better visibility into unhandled command types (I don't often expect there to be any).